charts/harbor-operator/README.md

```diff
@@ -30,7 +30,6 @@ Deploy Harbor Operator
 | nodeSelector | object | `{}` | Expects input structure as per specification <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#nodeselector-v1-core> For example: `[   {     "matchExpressions": [       {         "key": "kubernetes.io/e2e-az-name",         "operator": "In",         "values": [           "e2e-az1",           "e2e-az2"         ]       }     ]   } ]` |
 | podAnnotations | object | `{}` | Additional annotations to add to the controller Pods |
 | podLabels | object | `{}` | Additional labels to add to the controller Pods |
-| podSecurityContext | object | `{}` | Expects input structure as per specification <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritycontext-v1-core> For example: `{   "fsGroup": 2000,   "runAsUser": 1000,   "runAsNonRoot": true }` |
```

Comment:
Why did you remove this section?

Reply:
Because the operator manager default runs as a non-root user and runs as a user 65532, the `runAsUser` and `runAsNonRoot` is not needed now.

If the podSecurityContext not be removed, the customer needs to provide the podSecurityContext to run the operator in the restricted psp env.

Reply:
If the users want to override the run user, they can also provide this section `podSecurityContext`, right? @heww

Reply:
他已经在dockerfile中写死user id了，这个参数估计没有了

Reply:
他已经在dockerfile中写死user id了，这个参数估计没有了

Reply:
I tested that the pod also runs successes when providing another user id in the security context.